docker/pipelines_runner/pipelines_runner.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+set -euo pipefail
+
+# Copyright 2019 Google Inc.  All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+PROJECT_OPT="--project"
+TEMP_LOCATION_OPT="--temp_location"
+DOCKER_IMAGE_OPT="--docker_image"
+ZONES_OPT="--zones"
+COMMAND_OPT="--command"
+SERVICE_ACCOUNT_SCOPES="https://www.googleapis.com/auth/cloud-platform"
+
+#################################################
+# Parses arguments and does some sanity checking.
+# Arguments:
+#   It is expected that this is called with $@ of the main script.
+#################################################
+function parse_args {
```

Comment:
Consider using `getopt` to minimize the amount of checking you need to do here

Reply:
Done. Thanks! It is much neater!